### DISCUSSION TITLE
Summer 2022 MOOSE Workshop

### DISCUSSION THREAD

**Discussion by cticenhour** [idaholab/moose#20894] [News]
Idaho National Laboratory (INL) will be hosting a MOOSE workshop **June 6th and 7th, 2022**. Attendance to the workshop is free of charge, but you must register by filling out the Google form (https://bit.ly/3rVLuNL).

The workshop will run from **8:30 a.m. to 4:30 p.m MT on Mon. and Tues**. We will be hosting guest speakers for lunch time talks. After registering you can expect to receive additional details regarding the workshop via email as the dates approach. For more information about MOOSE see [mooseframework.inl.gov](http://mooseframework.inl.gov/) and if you have questions, please contact @permcody at [cody.permann@inl.gov](mailto:cody.permann@inl.gov).

This training covers the basic operation of MOOSE, focusing on the development of a multi-physics MOOSE-based application. The training aims to provide the basics of MOOSE such that attendees will have the necessary skills to begin developing a MOOSE-based application to meet their own needs.

In Person Attendance will be capped and INL Requires ~30 days to clear international staff members who would like to attend. Please register at your earliest convenience.

NOTE: We do not automatically send a confirmation of your registration but rest assured we've got it. We'll contact you with more details as the workshop approaches.

Once again, the registration form can be found at: https://bit.ly/3rVLuNL

**Comment by thomas-allard** [comment]
Is there an estimate on when we can watch the recording from the training last week?

**Reply by CGH20171006** [reply]
can we watch the recording?
